### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by damienbod**
small improvements to the ASP.NET Core project
- renaming to match default .NET styles
- moved the sln file 
- added a standard profile to match default ASP.NET Core
- removed blocking calls
- Added a note about user secrets in app settings
- Updated nuget packages
- namespace matches proj name

@mahoekst  @kalyankrishna1 Hope this is ok, greetings Damien

### CODE REVIEW

File: AspNetCoreVerifiableCredentials/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Make sure you copy the `ClientId`, `ClientSecret` and `TenantTd` you copied when
 
 1. Open a command prompt and run the following command:
 ```Powershell
-dotnet build "asp-net-core-api-idtokenhint.csproj" -c Debug -o .\bin\Debug\netcoreapp3.1
+dotnet build "AspNetCoreVerifiableCredentials.csproj" -c Debug -o .\bin\Debug\net5
```
Comment (mahoekst) [inline AspNetCoreVerifiableCredentials/README.md:97]: I think we still support code from .net3.1 @kalyankrishna1  might have an opinion on this as well

**Review by mahoekst — CHANGES_REQUESTED** [review]
I like the updates, it makes the code better and more readable. No objections from my side except the localhost binding in the project.

File: AspNetCoreVerifiableCredentials/Properties/launchSettings.json [hunk 2]
```diff
@@ -15,13 +15,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
-    "Verifiable_credentials_DotNet": {
+    "AspNetCoreVerifiableCredentials": {
       "commandName": "Project",
       "launchBrowser": true,
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
```
[the remark this answers is not in the document]
Comment (damienbod) [reply AspNetCoreVerifiableCredentials/Properties/launchSettings.json:21]: Hi @mahoekst, thanks for the review. I tested this using ngrok and this worked without problem. I was able to issue and verify without problem. I have not tried with docker. What does not work? What would you prefer?

Greetings Damien

File: AspNetCoreVerifiableCredentials/README.md (hunk 1 above)
Comment (damienbod) [reply AspNetCoreVerifiableCredentials/README.md:97]: The project is a .NET 5, so I updated this to match. If we revert back to 3.1, then maybe the proj as well? What do you recommend?

File: AspNetCoreVerifiableCredentials/Properties/launchSettings.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (damienbod) [reply AspNetCoreVerifiableCredentials/Properties/launchSettings.json:21]: This? 
```
"applicationUrl": "https://0.0.0.0:5001;http://0.0.0.0:5000",
```

**Comment by damienbod** [comment]
I updated the links in the docs due to this change:

https://github.com/MicrosoftDocs/azure-docs/pull/82437

**Pull request merged**